recipe/meta.yaml

```diff
@@ -58,17 +52,15 @@ outputs:
         - {{ pin_subpackage('gym-atari', exact=True) }}
         - {{ pin_subpackage('gym-box2d', exact=True) }}
         - {{ pin_subpackage('gym-classic_control', exact=True) }}
-        - {{ pin_subpackage('gym-mujoco', exact=True) }}
-        - {{ pin_subpackage('gym-robotics', exact=True) }}
+        # exclude some packages because thri dependency (mujoco) has
```

Comment:
I don't understand "thri dependency"

Reply:
LOL, that's a typo, should be `their`, gonna fix soon.

Reply:
also @h-vetinari are you sure mujoco license is forbidding? I've checked out their github, and it seems that it should be possible to install `mujoco-py` as it's MIT-licensed (which in turn would **not** work without MuJoCo, which is a paid thing, but that's a separate issue).

Reply:
That's a good point. I looked at the original staged-recipes PR, but couldn't find more rationale, except from (perhaps casual) reading of https://github.com/openai/mujoco-py#install-mujoco. But you're right, according to https://github.com/openai/mujoco-py/blob/master/LICENSE.md, we should be able to package it.

Double-checking this one with @oblute @scopatz

Reply:
That said, there's not much point of packaging, though, because it won't work until the user manually installs MoJoCo anyway, and at that point they may as well `conda install mojoco-py`....

Reply:
> [...] and at that point they may as well `conda install mojoco-py`....

Well, not unless we package it, right?

But I think we can consider changing this aspect to be the topic of a different PR.

Reply:
Opened #7 and moving forward with this PR, thanks! 👍